packages/mobile/src/tokens/selectors.ts

```diff
@@ -16,10 +17,11 @@ export const tokensByAddressSelector = createSelector(
         continue
       }
       const usdPrice = new BigNumber(storedState.usdPrice)
+      const tokenUsdPriceIsStale = storedState.priceFetchedAt < Date.now() - ONE_DAY_IN_MILLIS
```

Comment:
It seems we should surface this more explicitly with a dedicated config constant in `src/config.ts`. What do you think?